src/ResourceManager/ServiceFabric/Commands.ServiceFabric.Test/ScenarioTests/TestServiceFabric.ps1

```diff
@@ -106,7 +106,10 @@ function Test-AddAzureRmServiceFabricNodeType
 
 	$clusters = Get-AzureRmServiceFabricCluster -ClusterName $clusterName -ResourceGroupName $resourceGroupName
     $vmPassword = Get-VmPwd | ConvertTo-SecureString -Force -AsPlainText
-	$cluster = Add-AzureRmServiceFabricNodeType -Capacity 1 -VmUserName username -VmPassword $vmPassword -NodeType $newNodeTypeName -ClusterName $clusterName -ResourceGroupName $resourceGroupName -Verbose
+	# Added durability level, as default is Bronze and i get the following error 
+	# Remove-AzureRmServiceFabricNode : Cannot update node type with a durability level of Bronze, please update durability
+    # level to Silver or above first.
+	$cluster = Add-AzureRmServiceFabricNodeType -Capacity 1 -DurabilityLevel Silver -VmUserName username -VmPassword $vmPassword -NodeType $newNodeTypeName -ClusterName $clusterName -ResourceGroupName $resourceGroupName -Verbose
```

Comment:
This is what Test-UpdateAzureRmServiceFabricDurability is doing. You need to run that test after adding the node type. You can leave this for now and I will make the order of test execution explicit in the file with my next change.

Reply:
@juhacket  sounds good to me...